### PR TITLE
Fix automatic startup of mavlink on mRo Pixracer Wifi Bridge port

### PR DESCRIPTION
**Pull request by marcojob**
**Describe problem solved by this pull request**
Using master and a mRo pixracer (fmu-v4), mavlink does not automatically launch on the dedicated wifi bridge (ESP8266) port. 

**Describe your solution**
Revert removal of rc.board_mavlink for fmu-v4 in order to fix automatic startup of telemetry link for mRo pixracer

**Test data / coverage**
Tested with mRo pixracer

**Additional context**
At some point there was an if statement for mavlink for the teal one airframe on this board, but since this does not exist anymore, I removed it.

### CODE REVIEW

File: boards/px4/fmu-v4/init/rc.board_mavlink [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# PX4 FMUv4 specific board MAVLink startup script.
+#------------------------------------------------------------------------------
+
+# Start MAVLink on the USB port
+mavlink start -d /dev/ttyACM0
```
Comment (dagar) [inline boards/px4/fmu-v4/init/rc.board_mavlink:7]: This is done automatically now based on VBUS.

File: boards/px4/fmu-v4/init/rc.board_mavlink [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+# Pixracer: start MAVLink on Wifi (ESP8266 port)
+mavlink start -r 20000 -b 921600 -d /dev/ttyS0
```
Comment (dagar) [inline boards/px4/fmu-v4/init/rc.board_mavlink:10]: Did you try configuring this via parameters? https://github.com/PX4/PX4-Autopilot/blob/cbc37f9fcda4a3f01112f542c3940dbe3ea46ea1/boards/px4/fmu-v4/default.px4board#L7

Maybe we need to be setting appropriate default parameter values instead.

File: boards/px4/fmu-v4/init/rc.board_mavlink (hunk 1 above)
Comment (marcojob) [reply boards/px4/fmu-v4/init/rc.board_mavlink:7]: Ok, will remove

File: boards/px4/fmu-v4/init/rc.board_mavlink (hunk 2 above)
Comment (marcojob) [reply boards/px4/fmu-v4/init/rc.board_mavlink:10]: Not sure if I understand, do you mean something like this:
```suggestion
mavlink start -r 20000 -b 921600 -d {CONFIG_BOARD_SERIAL_WIFI}
```
I did not find this port being defined as a parameter, but I could add it

**Comment by dagar** [comment]
@marcojob 

It's in the param metadata per mavlink instance (or other serial module).

Screenshot from 2022-04-25 13-17-34

Then you set the baudrate with `SER_WIFI_BAUD`.
Screenshot from 2022-04-25 13-18-24

We could consider setting these parameters by default, but we need to be careful with free memory availability on older boards.

**Pull request closed without merging**